shared/src/commonMain/resources/MR/base/plurals.xml

```diff
@@ -12,6 +12,14 @@
         <item quantity="one">%d hour</item>
         <item quantity="other">%d hours</item>
     </plural>
+    <plural name="minutes_short">
+        <item quantity="one">%d m.</item>
+        <item quantity="other">%d m.</item>
+    </plural>
+    <plural name="hours_short">
+        <item quantity="one">%d h.</item>
+        <item quantity="other">%d h.</item>
+    </plural>
```

Comment:
If we have the same vales for `one` and `other`. Maybe we can use only `other` item or just move that resource to `strings.xml`

Reply:
@vladkash 
You are right, in this case we case use just a string not plurals